includes/class-wc-stripe-webhook-handler.php

```diff
@@ -596,6 +596,71 @@ public function process_webhook_refund( $notification ) {
 		}
 	}
 
+	/**
+	 * Process a refund update.
+	 *
+	 * @param object $notification
+	 */
+	public function process_webhook_refund_updated( $notification ) {
+		$refund_object = $notification->data->object;
+		$order         = WC_Stripe_Helper::get_order_by_charge_id( $refund_object->charge );
+
+		if ( ! $order ) {
+			WC_Stripe_Logger::log( 'Could not find order to update refund via charge ID: ' . $refund_object->charge );
+			return;
+		}
+
+		$order_id = $order->get_id();
+
+		if ( 'stripe' === $order->get_payment_method() ) {
+			$charge    = $order->get_transaction_id();
+			$refund_id = $order->get_meta( '_stripe_refund_id' );
+
+			$amount = wc_price( $refund_object->amount / 100 );
+			if ( in_array( strtolower( $order->get_currency() ), WC_Stripe_Helper::no_decimal_currencies(), true ) ) {
+				$amount = wc_price( $refund_object->amount );
+			}
```

Comment:
```suggestion
			$amount = wc_price( $refund_object->amount / 100, [ 'currency' => $order->get_currency() ] );
			if ( in_array( strtolower( $order->get_currency() ), WC_Stripe_Helper::no_decimal_currencies(), true ) ) {
				$amount = wc_price( $refund_object->amount, [ 'currency' => $order->get_currency() ] );
			}
```
`wc_price` defaults to using `get_woocommerce_currency`, which returns the current default currency, for the currency to use when formatting the price: should we pass in the currency on the order, just in case it differs from the default? I think this would only be applicable with multi-currency, but I could see this happening.

Reply:
Well if we do that we'll need to fix it in `process_webhook_refund()` too.